prometheus_client/multiprocess.py

```diff
@@ -155,9 +167,12 @@ def merge(files, accumulate=True):
             metric.samples = [Sample(name_, dict(labels), value) for (name_, labels), value in samples.items()]
         return metrics.values()
 
-    def collect(self):
-        files = glob.glob(os.path.join(self._path, '*.db'))
-        return self.merge(files, accumulate=True)
+    def collect(self, blocking=True):
+        # blocking is used for testing purposes
```

Comment:
This would be clearer to me if it was "non-blocking is used for testing purposes"